client/src/modules/multiple_payroll/multiple_payroll.js

```diff
@@ -50,11 +52,17 @@ function MultiplePayrollController(
     displayName : 'FORM.LABELS.NET_SALARY',
     headerCellFilter : 'translate',
     cellFilter : 'currency:row.entity.currency_id',
+    aggregationType : uiGridConstants.aggregationTypes.sum,
+    aggregationHideLabel : true,
+    footerCellFilter : 'currency:'.concat(Session.enterprise.currency_id),
```

Comment:
Can you pay employees in multiple currencies?  What happens if the grid contains FC + USD?

Reply:
You're right, I'm going to delete it